index2.html

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
There's a much easier, Jekyll way to do this. Here's an example: https://github.com/jekyll/example/blob/gh-pages/about.md

Reply:
I can not get that way to work. Will convene with you in the future and fix later.